server/plugin.go

```diff
@@ -33,6 +33,8 @@ const (
 	SETTING_OFF           = "off"
 )
 
+var emptySiteURLErr = errors.New("Unable to connect to GitLab")
```

Comment:
That error message doesn't make clear what a user/admin should do to fix it. Maybe something like:
```suggestion
var emptySiteURLErr = errors.New("SiteURL is not set. Please set it and restart the plugin.")
```